### PR TITLE
Fix flake8 issues on asv/commands folder

### PR DESCRIPTION
**Pull request by LucyJimenez**
xref https://github.com/airspeed-velocity/asv/issues/1065

Clean out all the files to follow Flake8 standards on the asv/commands folder.

### CODE REVIEW

File: asv/commands/check.py [hunk 1]
```diff
@@ -42,4 +40,5 @@ def run(cls, conf, env_spec=None):
         commit_hashes = []  # use fallback defaults
         benchmarks = Benchmarks.discover(conf, repo, environments, commit_hashes,
                                          check=True)
+        benchmarks
```
Comment (datapythonista) [inline asv/commands/check.py:43]: What's the idea here?

File: asv/commands/find.py [hunk 2]
```diff
@@ -100,7 +100,8 @@ def run(cls, conf, range_spec, bench, invert=False, show_stderr=False, parallel=
         elif len(benchmarks) > 1:
             exact_matches = benchmarks.filter_out([x for x in benchmarks if x != bench])
             if len(exact_matches) == 1:
-                log.warning("'{0}' matches more than one benchmark, using exact match".format(bench))
+                log.warning("'{0}' matches more than one benchmark,"
```
Comment (datapythonista) [inline asv/commands/find.py:103]: There is a missing space after the comma

File: asv/commands/run.py [hunk 3]
```diff
@@ -214,8 +214,8 @@ def run(cls, conf, range_spec=None, steps=None, date_period=None,
             if dry_run:
                 raise util.UserError("--interleave-rounds and --dry-run cannot be used together")
             if has_existing_env:
-                raise util.UserError("--interleave-rounds cannot be used with existing environment "
-                                     "(or python=same)")
+                raise util.UserError("--interleave-rounds cannot be used with"
```
Comment (datapythonista) [inline asv/commands/run.py:217]: missing space

**Comment by LucyJimenez** [comment]
I'm going to slip this PR on small tasks to review the errors easily.

**Review by datapythonista — APPROVED** [review]
Thanks @LucyJimenez for the clean up

**Pull request merged**